internal/appinfo/appinfo.go

```diff
@@ -17,14 +17,14 @@ type AppInfo struct {
 func TrayApp() *AppInfo {
 	return &AppInfo{
 		constants.TrayAppName,
-		absolute(filepath.Join(filepath.Dir(os.Args[0]), "state-tray") + osutils.ExeExt),
+		exePath(filepath.Join(filepath.Dir(os.Args[0]), "state-tray") + osutils.ExeExt),
```

Comment:
I noticed that this needs to look like this now:

 ```
exePath("state-tray")
```

Reply:
Thanks, just noticed that also -.-